### PR TITLE
Release 2022.06.2

### PR DESCRIPTION
**Pull request by brianduffield-okta**
## Description:
- **What's changed?** Release branch contains one dev doc update and the addition of weekly API and OIE API Release Notes.
- **Is this PR related to a Monolith release?** 2022.06.2

### Resolves:

* [OKTA-507791](https://oktainc.atlassian.net/browse/OKTA-507791)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md [hunk 1]
```diff
@@ -6,6 +6,26 @@ title: Okta Identity Engine API Products release notes 2022
...
+
+| Change | Expected in Preview Orgs |
+|--------------------------------------------------------------------------|--------------------------|
+| [Bugs fixed in 2022.06.2](#bugs-fixed-in-2022-06-2)                      | June 23, 2022            |
+
+#### Bugs fixed in 2022.06.2
+
+* When an OAuth 2.0 client was created with a missing JWKS RSA modulus value (`n` parameter), the JWKS validation failed. (OKTA-424664)
+
+* Making a POST reset factors request to `/api/v1/users/${userId}/lifecycle/reset_factors` didn't reset the enrolled phone authenticator. (OKTA-463900)
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md:19]:
```suggestion
* A POST reset factors request to `/api/v1/users/${userId}/lifecycle/reset_factors` didn't reset the enrolled phone authenticator. (OKTA-463900)
```

File: packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md [hunk 2]
```diff
@@ -6,6 +6,26 @@ title: Okta Identity Engine API Products release notes 2022
...
+
+* When the `maxSessionLifetimeMinutes` property of the Policy API Signon Session object was set, the value wasn't enforced. (OKTA-480442)
+
+* Using the JSON Web Key API or the Client Credentials API to manage client credentials didn't fire System Log events. (OKTA-494619)
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md:23]:
```suggestion
* When the JSON Web Key API or the Client Credentials API was used to manage client credentials, the event wasn't triggered in the System Log. (OKTA-494619)
```

File: packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md [hunk 3]
```diff
@@ -6,6 +6,26 @@ title: Okta Identity Engine API Products release notes 2022
...
+
+* When the Progressive Profiling feature was enabled, calling the `/introspect` endpoint changed the state handle incorrectly. (OKTA-502233)
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md:25]:
```suggestion
* When the Progressive Profiling feature was enabled, a request to the `/introspect` endpoint changed the state handle incorrectly. (OKTA-502233)
```

File: packages/@okta/vuepress-site/docs/release-notes/2022/index.md [hunk 4]
```diff
@@ -4,6 +4,25 @@ title: Okta API Products release notes 2022
...
 ## June
 
+### Weekly release 2022.06.2
+
+| Change | Expected in Preview Orgs |
+|--------------------------------------------------------------------------|--------------------------|
+| [Reset Factors endpoint includes new optional request parameters](#bugs-fixed-in-2022-06-2)                      | June 23, 2022            |
+| [Bugs fixed in 2022.06.2](#bugs-fixed-in-2022-06-2)                      | June 23, 2022            |
+
+#### Reset Factors endpoint includes new optional request parameter
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2022/index.md:14]:
```suggestion
#### Reset Factors endpoint includes a new optional request parameter
```

File: packages/@okta/vuepress-site/docs/release-notes/2022/index.md [hunk 5]
```diff
@@ -4,6 +4,25 @@ title: Okta API Products release notes 2022
 
 ## June
 
+### Weekly release 2022.06.2
+
+| Change | Expected in Preview Orgs |
+|--------------------------------------------------------------------------|--------------------------|
+| [Reset Factors endpoint includes new optional request parameters](#bugs-fixed-in-2022-06-2)                      | June 23, 2022            |
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2022/index.md:11]:
```suggestion
| [Reset Factors endpoint includes new a optional request parameters](#reset-factors-endpoint-includes-new-a-optional-request-parameters)                      | June 23, 2022            |
```

File: packages/@okta/vuepress-site/docs/release-notes/2022/index.md [hunk 6]
```diff
@@ -4,6 +4,25 @@ title: Okta API Products release notes 2022
...
+| [Bugs fixed in 2022.06.2](#bugs-fixed-in-2022-06-2)                      | June 23, 2022            |
+
+#### Reset Factors endpoint includes new optional request parameter
+
+The `/reset_factors` endpoint has a new optional request parameter for the Reset Factor lifecycle operation. You can now remove the phone factor (for example: SMS/Voice) as both a recovery method and a factor with one call by setting the `removeRecoveryEnrollment` parameter to true when making a POST request to the `/reset_factors` endpoint (`/users/${userId}/lifecycle/reset_factors`). <!--OKTA-500168-->
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2022/index.md:16]:
```suggestion
The `/reset_factors` endpoint has a new optional request parameter for the [Reset Factor lifecycle operation](/docs/reference/api/users/#reset-factors). You can now remove the phone factor (for example: SMS/Voice) as both a recovery method and a factor by setting the `removeRecoveryEnrollment` parameter to true when making a POST request to the `/reset_factors` endpoint (`/users/${userId}/lifecycle/reset_factors`). <!--OKTA-500168-->
```
I'm thinking that there should be a link to the Reset Factor API operation? You can remove the link if you think it's not necessary.

File: packages/@okta/vuepress-site/docs/release-notes/2022/index.md [hunk 7]
```diff
@@ -4,6 +4,25 @@ title: Okta API Products release notes 2022
...
+
+#### Bugs fixed in 2022.06.2
+
+* When an OAuth 2.0 client was created with a missing JWKS RSA modulus value (`n` parameter), the JWKS validation failed. (OKTA-424664)
+
+* Using the JSON Web Key API or the Client Credentials API to manage client credentials didn't fire System Log events. (OKTA-494619)
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2022/index.md:22]:
```suggestion
* When the JSON Web Key API or the Client Credentials API was used to manage client credentials, the event wasn't triggered in the System Log. (OKTA-494619)
```

**Review by vanngo-okta — CHANGES_REQUESTED** [review]
A few nits. I wasn't able to build and test the reference links.

File: packages/@okta/vuepress-site/docs/release-notes/2022/index.md (hunk 6 above)
Comment (brianduffield-okta) [reply packages/@okta/vuepress-site/docs/release-notes/2022/index.md:16]: I agree, for the feature update, to add the link.

**Review by vanngo-okta — APPROVED** [review]
LGTM! 🚀

**Pull request merged**